src/pages/HomePage.tsx

```diff
@@ -0,0 +1 @@
+export const HomePage = () => <h1 className="title">Home page</h1>;
```

Comment:
Let's use the container div to wrap our pages to avoid using react fragments in some cases

```suggestion
export const HomePage = () => (
  <div className="container"
    <h1 className="title">Home page</h1>
  </div>
 );
```